Lib/test/test_zlib.py

```diff
@@ -165,6 +165,7 @@ def test_speech(self):
         x = zlib.compress(HAMLET_SCENE)
         self.assertEqual(zlib.decompress(x), HAMLET_SCENE)
 
+    @support.cpython_only
```

Comment:
It's only the `TypeError` check that should be skipped, right? Not the whole test.